tests/fips/openssl/dirmngr_daemon.pm

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test dirmngr daemon and valid/revoke certificate
+#
+# Maintainer: Ben Chou <bchou@suse.com>
+# Tags: poo#52430, poo#52937
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub dirmngr_daemon {
+
+    # Create dirmngr required testing folder
+    assert_script_run('mkdir -p /etc/dirmngr');
+    assert_script_run('mkdir -p /etc/dirmngr/trusted-certs');
+    assert_script_run('mkdir -p /var/run/dirmngr');
+    assert_script_run('mkdir -p /var/log/dirmngr');
```

Comment:
You could also spare a line here with a [bash expansion](https://www.gnu.org/software/bash/manual/bash.html#Brace-Expansion):
` # mkdir /var/{log,run}/dirmngr`
` # ls /var/{log,run}/dirmngr -d`
`/var/log/dirmngr  /var/run/dirmngr`
no need for the -p here, those directories exist by default.

Reply:
done, thanks.